src/css3/tokenizer.rs

```diff
@@ -91,13 +91,13 @@ mod test {
         let mut tokenizer = Tokenizer::default();
         tokenizer.init("123 -ident-test-1");
 
-        assert!(!tokenizer.is_eof());
-        assert!(tokenizer.has_more_tokens());
+        assert_eq!(tokenizer.is_eof(), false);
+        assert_eq!(tokenizer.has_more_tokens(), true);
```

Comment:
Can you run `make fix` on this PR?  I'm wondering whether these changes will go back to what was there before.